sql/wsrep_schema.cc

```diff
@@ -1535,3 +1536,32 @@ int Wsrep_schema::recover_sr_transactions(THD *orig_thd)
 out:
   DBUG_RETURN(ret);
 }
+
+int Wsrep_schema::store_gtid_event(THD* orig_thd,
```

Comment:
Why is this method necessary? The GTID state seems to be recorded from `Xid_apply_log_event::do_apply_event()`, why is that codepath not effective for Galera?

Reply:
I think it is because XID event is not received by applier.